app/controllers/api/v0/base_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Api
+  module V0
+    class BaseController < ApplicationController
+      def list_order_item
+        item = OrderItem.where('id = ? and order_id = ?',
+                               params['order_item_id'], params['order_id']).first
+        render json: item.to_hash
+      end
+
+      def list_order_items
+        render json: OrderItem.where(:order_id => params['order_id']).collect(&:to_hash)
```

Comment:
Again, no need to enumerate this. See https://github.com/ManageIQ/service_portal-api/pull/42/files#r233591226
And also, Strong Params